stable/phabricator/README.md

```diff
@@ -112,3 +112,14 @@ The [Bitnami Phabricator](https://github.com/bitnami/bitnami-docker-phabricator)
 
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
 See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
+
+## Ingress With Reverse Proxy And Kube Lego
+
+You can define a custom ingress following the example config in values.yaml 
+
+`helm install stable/phabricator/ --name my-release --set phabricatorHost=example.com`
+
+Everything looks great but requests over https will cause asset requests to fail. Assuming you want to use HTTPS/TLS you will need to set the base-uri to an https schema.
+
+`kubectl exec <pod name> /opt/bitnami/phabricator/bin/config set phabricator.base-uri https://example.com`
```

Comment:
If possible try to use an expression that renders the appropriate pod name instead of using the `<pod name>` placeholder.